Sources/TensorFlow/Layers/Normalization.swift

```diff
@@ -224,3 +244,166 @@ public struct LayerNorm<Scalar: TensorFlowFloatingPoint>: Layer {
     return (input - moments.mean) * inv + offset
   }
 }
+
+/// A layer that applies group normalization over a mini-batch of inputs.
+///
+/// Reference: [Group Normalization](https://arxiv.org/abs/1803.08494).
+@frozen
+public struct GroupNorm<Scalar: TensorFlowFloatingPoint>: Layer {
+  /// The offset value, also known as beta.
+  public var offset: Tensor<Scalar>
+  /// The scale value, also known as gamma.
+  public var scale: Tensor<Scalar>
+  /// The number of groups.
+  @noDerivative public let groups: Int
+  /// The axis.
+  @noDerivative public let axis: Int
+  /// The variance epsilon value.
+  @noDerivative public let epsilon: Scalar
+
+  /// Creates a group normalization layer.
```

Comment:
This summary isn't adding any useful information to the code.  We know it creates an instance of `GroupNorm`— that's given by the language—which is documented as a group normalization layer.  Usually we want to say something about the characteristics of the instance created, and how it relates to the arguments passed.  Sometimes that's hard, and we might not be able to do better than “Creates an instance having the given attributes,” but it's good to try.  And if that's all we can say, we should have a `- Parameters:` clause that documents each of the parameters, and what they mean.  I realize that may end up repeating the doc comments you've got on stored properties, but the stored properties don't show up to the person looking at the init in the API documentation or the quick help in an IDE.

Reply:
Also, the preconditions should be noted in the doc comment as well